vnext/ReactUWP/Views/ViewPanel.h

```diff
@@ -50,40 +48,58 @@ struct ViewPanel : winrt::Windows::UI::Xaml::Controls::PanelT<ViewPanel>
   bool ClipChildren() { return winrt::unbox_value<bool>(GetValue(ClipChildrenProperty())); }
   void ClipChildren(bool value);
 
-public:
   // ViewPanel Properties
-  static winrt::Windows::UI::Xaml::DependencyProperty BorderBrushProperty();
   static winrt::Windows::UI::Xaml::DependencyProperty BorderThicknessProperty();
+  static winrt::Windows::UI::Xaml::DependencyProperty BorderBrushProperty();
   static winrt::Windows::UI::Xaml::DependencyProperty CornerRadiusProperty();
-  static winrt::Windows::UI::Xaml::DependencyProperty BackgroundProperty();
   static winrt::Windows::UI::Xaml::DependencyProperty ClipChildrenProperty();
```

Comment:
Why is this one removed? I think the new code will mess up the desired behavior. This property is separate from Panel::BackgroundProperty because it can be applied to either an outer Border element or to the Panel itself. This is a decision made in FinalizeProperties starting on line 332.

Reply:
Working on a fix, will rename the property so it doesn't conflict with Panel::BackgroundProperty.

Reply:
REsolved.